tests/ops/test_qubit_ops.py

```diff
@@ -989,9 +989,9 @@ def circuit(theta):
             return qml.expval(qml.PauliZ(0))
 
         res = circuit(angle)
-        gradient = np.squeeze(circuit.jacobian(angle))
+        gradient = np.squeeze(qml.grad(circuit)(angle))
 
-        assert gradient == 0.5 * (circuit(angle + np.pi / 2) - circuit(angle - np.pi / 2))
+        assert gradient == pytest.approx(0.5 * (circuit(angle + np.pi / 2) - circuit(angle - np.pi / 2)), abs=tol)
```

Comment:
Any difference compared to using `np.allclose`?

Reply:
Not sure! These tests already used `pytest.approx`, and I wanted to limit superfluous/out of scope changes.